css/lean.css

```diff
@@ -1,4 +1,13 @@
 @charset "UTF-8";
+
```

Comment:
These changes should be made to `scss/lean.scss` file, since `css/lean.css` should be generated by `sass`: see https://github.com/leanprover-community/leanprover-community.github.io#building